code/modules/telesci/gps.dm

```diff
@@ -119,6 +119,7 @@ var/list/SPS_list = list()
 	gpstag = "SEC0"
 
 /obj/item/device/gps/secure/New()
+	..()
```

Comment:
You can't just do this, it didn't inherit new because it didn't want the behaviour from gps/New